lib/zoneinfo.ex

```diff
@@ -78,6 +78,16 @@ defmodule Zoneinfo do
   @spec get_metadata(String.t()) :: {:ok, Zoneinfo.Meta.t()} | {:error, atom()}
   defdelegate get_metadata(time_zone), to: Zoneinfo.Cache, as: :meta
 
+  @doc """
+  Return the time zone database version
+
+  This attempts to read `<tzpath>/version` and returns `:unknown` if non-existent.
```

Comment:
I think this will normally return `:unknown` since the file `version` doesn't exist normally. The place that you copied above is only for unit tests.

I've never found any consistency for finding the database version. On MacOS, there's a file called `+VERSION` that has it. On Ubuntu, the `tzdata.zi` file lists it as a comment.

Reply:
Rereading your comments, I'm not sure if you intended this to be only for unit tests. If so, how about moving this function out of the public API?